Tests/TSFCASFileTreeTests/FileTreeImportExportTests.swift

```diff
@@ -64,7 +64,13 @@ class ImportExportTests: XCTestCase {
             // Export the results.
             let tmpdir2 = dir.appending(component: "second")
             try fs.createDirectory(tmpdir2)
-            try LLBCASFileTree.export(id, from: db, to: tmpdir2, ctx).wait()
+            try LLBCASFileTree.export(
+                id,
+                from: db,
+                to: tmpdir2,
+                stats: LLBCASFileTree.ExportProgressStatsInt64(),
+                ctx
+            ).wait()
```

Comment:
Should move the tests to get() too

Reply:
We should use `.get()` if the test func is `async throws` but if the test func is just `throws` we can of course leave it. The `.wait()` doesn't hurt, in fact it can help a tiny little bit to make sure we're not trying to use `DispatchQueue.main` or the main runloop. That'd be very bad for server software but occasionally a `DispatchQueue.main.async { ... }` sneaks in somewhere.